# HG changeset patch
# Parent bae4882fd1a3e5620ad8afebc024779a39b3f4e7
Various configuration files.

Most notibly, the cached results of

./configure --disable-seabios --prefix="/usr" --disable-stubdom

diff -r bae4882fd1a3 .config
--- /dev/null
+++ b/.config
@@ -0,0 +1,8 @@
+# Do no build Qemu as part of Xen
+CONFIG_IOEMU := n
+
+# Perform a debug build for trunk
+debug := y
+# debug_symbols := y
+
+LIBXL_BLKTAP := n
diff -r bae4882fd1a3 config/Docs.mk
--- /dev/null
+++ b/config/Docs.mk
@@ -0,0 +1,14 @@
+# Prefix and install folder
+prefix              := /usr
+PREFIX              := $(prefix)
+exec_prefix         := ${prefix}
+libdir              := ${exec_prefix}/lib
+LIBDIR              := $(libdir)
+
+# Tools
+FIG2DEV             := 
+POD2MAN             := /usr/bin/pod2man
+POD2HTML            := /usr/bin/pod2html
+POD2TEXT            := /usr/bin/pod2text
+MARKDOWN            := 
+PERL                := /usr/bin/perl
diff -r bae4882fd1a3 config/Stubdom.mk
--- /dev/null
+++ b/config/Stubdom.mk
@@ -0,0 +1,45 @@
+# Prefix and install folder
+prefix              := /usr/local
+PREFIX              := $(prefix)
+exec_prefix         := ${prefix}
+libdir              := ${exec_prefix}/lib
+LIBDIR              := $(libdir)
+
+# Path Programs
+CMAKE               := /usr/bin/cmake
+FETCHER             := /usr/bin/wget -c -O
+
+# A debug build of stubdom? //FIXME: Someone make this do something
+debug               := y
+vtpm = y
+
+STUBDOM_TARGETS     :=  ioemu grub xenstore vtpm vtpmmgr
+STUBDOM_BUILD       :=  ioemu-stubdom pv-grub xenstore-stubdom vtpm-stubdom vtpmmgr-stubdom
+STUBDOM_INSTALL     :=  install-ioemu install-grub install-xenstore install-vtpm install-vtpmmgr
+
+ZLIB_VERSION        := 1.2.3
+ZLIB_URL            := $(XEN_EXTFILES_URL)
+
+LIBPCI_VERSION      := 2.2.9
+LIBPCI_URL          := $(XEN_EXTFILES_URL)
+
+NEWLIB_VERSION      := 1.16.0
+NEWLIB_URL          := $(XEN_EXTFILES_URL)
+
+LWIP_VERSION        := 1.3.0
+LWIP_URL            := $(XEN_EXTFILES_URL)
+
+GRUB_VERSION        := 0.97
+GRUB_URL            := $(XEN_EXTFILES_URL)
+
+OCAML_VERSION       := 3.11.0
+OCAML_URL           := http://caml.inria.fr/pub/distrib/ocaml-3.11
+
+GMP_VERSION         := 4.3.2
+GMP_URL             := $(XEN_EXTFILES_URL)
+
+POLARSSL_VERSION    := 1.1.4
+POLARSSL_URL        := $(XEN_EXTFILES_URL)
+
+TPMEMU_VERSION      := 0.7.4
+TPMEMU_URL          := $(XEN_EXTFILES_URL)
diff -r bae4882fd1a3 config/Tools.mk
--- /dev/null
+++ b/config/Tools.mk
@@ -0,0 +1,64 @@
+# Prefix and install folder
+prefix              := /usr
+PREFIX              := $(prefix)
+exec_prefix         := ${prefix}
+libdir              := ${exec_prefix}/lib
+LIBDIR              := $(libdir)
+
+# A debug build of tools?
+debug               := y
+
+# Tools path
+BISON               := /usr/bin/bison
+FLEX                := /usr/bin/flex
+PYTHON              := python
+PYTHON_PATH         := /usr/bin/python
+PERL                := /usr/bin/perl
+CURL_CONFIG         := 
+XML2_CONFIG         := 
+BASH                := /bin/sh
+XGETTTEXT           := /usr/bin/xgettext
+AS86                := /usr/bin/as86
+LD86                := /usr/bin/ld86
+BCC                 := /usr/bin/bcc
+IASL                := /usr/bin/iasl
+FETCHER             := /usr/bin/wget -c -O
+
+# Extra folder for libs/includes
+PREPEND_INCLUDES    := 
+PREPEND_LIB         := 
+APPEND_INCLUDES     := 
+APPEND_LIB          := 
+
+PTHREAD_CFLAGS      := -pthread
+PTHREAD_LDFLAGS     := -pthread
+PTHREAD_LIBS        := 
+
+PTYFUNCS_LIBS       := -lutil
+
+# Download GIT repositories via HTTP or GIT's own protocol?
+# GIT's protocol is faster and more robust, when it works at all (firewalls
+# may block it). We make it the default, but if your GIT repository downloads
+# fail or hang, please specify GIT_HTTP=y in your environment.
+GIT_HTTP            := n
+
+# Optional components
+XENSTAT_XENTOP      := y
+LIBXENAPI_BINDINGS  := n
+OCAML_TOOLS         := y
+FLASK_POLICY        := n
+CONFIG_MINITERM     := n
+CONFIG_LOMOUNT      := n
+CONFIG_OVMF         := n
+CONFIG_ROMBIOS      := y
+CONFIG_SEABIOS      := n
+
+#System options
+CONFIG_SYSTEM_LIBAIO:= y
+ZLIB                :=  -DHAVE_BZLIB -lbz2
+CONFIG_LIBICONV     := n
+CONFIG_GCRYPT       := y
+EXTFS_LIBS          := -lext4fs
+CURSES_LIBS         := -lncurses
+
+FILE_OFFSET_BITS    := 64
diff -r bae4882fd1a3 config/Toplevel.mk
--- /dev/null
+++ b/config/Toplevel.mk
@@ -0,0 +1,1 @@
+SUBSYSTEMS               :=  xen kernels tools docs
diff -r bae4882fd1a3 tools/config.h
--- /dev/null
+++ b/tools/config.h
@@ -0,0 +1,80 @@
+/* config.h.  Generated from config.h.in by configure.  */
+/* config.h.in.  Generated from configure.ac by autoheader.  */
+
+/* Define to 1 if you have the <inttypes.h> header file. */
+#define HAVE_INTTYPES_H 1
+
+/* Define to 1 if you have the `crypto' library (-lcrypto). */
+#define HAVE_LIBCRYPTO 1
+
+/* Define to 1 if you have the `yajl' library (-lyajl). */
+#define HAVE_LIBYAJL 1
+
+/* Define to 1 if you have the `z' library (-lz). */
+#define HAVE_LIBZ 1
+
+/* Define to 1 if you have the <memory.h> header file. */
+#define HAVE_MEMORY_H 1
+
+/* Define to 1 if you have the <stdint.h> header file. */
+#define HAVE_STDINT_H 1
+
+/* Define to 1 if you have the <stdlib.h> header file. */
+#define HAVE_STDLIB_H 1
+
+/* Define to 1 if you have the <strings.h> header file. */
+#define HAVE_STRINGS_H 1
+
+/* Define to 1 if you have the <string.h> header file. */
+#define HAVE_STRING_H 1
+
+/* Define to 1 if you have the <sys/eventfd.h> header file. */
+/* #undef HAVE_SYS_EVENTFD_H */
+
+/* Define to 1 if you have the <sys/stat.h> header file. */
+#define HAVE_SYS_STAT_H 1
+
+/* Define to 1 if you have the <sys/types.h> header file. */
+#define HAVE_SYS_TYPES_H 1
+
+/* Define to 1 if you have the <unistd.h> header file. */
+#define HAVE_UNISTD_H 1
+
+/* Define to 1 if you have the <yajl/yajl_version.h> header file. */
+#define HAVE_YAJL_YAJL_VERSION_H 1
+
+/* Define curses header to use */
+#define INCLUDE_CURSES_H <ncurses.h>
+
+/* Define extfs header to use */
+#define INCLUDE_EXTFS_H <ext4fs/ext2fs.h>
+
+/* libutil header file name */
+/* #undef INCLUDE_LIBUTIL_H */
+
+/* Define to the address where bug reports for this package should be sent. */
+#define PACKAGE_BUGREPORT "xen-devel@lists.xen.org"
+
+/* Define to the full name of this package. */
+#define PACKAGE_NAME "Xen Hypervisor Tools"
+
+/* Define to the full name and version of this package. */
+#define PACKAGE_STRING "Xen Hypervisor Tools 4.3"
+
+/* Define to the one symbol short name of this package. */
+#define PACKAGE_TARNAME "xen"
+
+/* Define to the home page for this package. */
+#define PACKAGE_URL "http://www.xen.org/"
+
+/* Define to the version of this package. */
+#define PACKAGE_VERSION "4.3"
+
+/* Define to 1 if you have the ANSI C header files. */
+#define STDC_HEADERS 1
+
+/* Number of bits in a file offset, on hosts where this is settable. */
+#define _FILE_OFFSET_BITS 64
+
+/* Define for large files, on AIX-style hosts. */
+/* #undef _LARGE_FILES */
